sigstore/models.py

```diff
@@ -328,6 +335,62 @@ def _verify(self, keyring: RekorKeyring) -> None:
             )
 
 
+class TimestampVerificationData:
+    """
+    Represents a TimestampVerificationData structure.
```

Comment:
Let's make this `@private` for now so we can iterate on it, if necessary.

```suggestion
    Represents a TimestampVerificationData structure.
    
    @private
```